include/llvm/Transforms/IPO/StructFieldCacheAnalysis.h

```diff
@@ -0,0 +1,30 @@
+// llvm/Tranforms/IPO/StructFieldCacheAnalysis.h - Performs Cache-Aware Structure Analysis- C++ -*-===//
```

Comment:
I suggest we leave the code here if there's no better place. I put the code here was because it is an inter-procedure analysis. But I know it's not an IPO and doesn't change the code.